components/Header/index.tsx

```diff
@@ -29,144 +30,118 @@ interface HeaderProps {
   clear?: () => Promise<void>
 }
 
-interface State {
-  chosenRoute: string
-}
-
-class Header extends React.Component<HeaderProps, State> {
-  state: State = {
-    chosenRoute: '/',
-  }
-
-  connectAccount = async () => {
+const Header: React.FC<Props> = (props: Props) => {
+  const connectAccount = async () => {
     try {
-      await this.props.ensureAuthed!()
+      await props.ensureAuthed!()
     } catch (e) {
       console.error(`authentication failed with Error ${e}`)
     }
   }
 
-  onRouteClick = (item: MenuItem) => {
-    this.setState({ chosenRoute: item.route })
+  const onRouteClick = (item: MenuItem) => {
+    // setChosenRoute(item.route)
```

Comment:
Why is this commented out?

Reply:
Oops, that must have been a mistake, good catch!